sh_scrapy/extension.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, crawler):
 
         self.crawler = crawler
         self._write_item = hsref.job.items.write
-        self.exporter = PythonItemExporter()
+        self.exporter = PythonItemExporter(binary=False)
```

Comment:
This change is important, related with that the exporter converts items keys to bytes in Python 3, this leads to exceptions `keys must be a string` in python-hubstorage when writing items. See [the issue](https://github.com/scrapy/scrapy/issues/1080) and related [PR](https://github.com/scrapy/scrapy/pull/1499) for the details.